### PR TITLE
Psychedelic twist on Greek version

### PR DESCRIPTION
**Pull request by pati-ni**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2570)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: greek/offer_marshmallow_a_cigar/offer_marshmallow_a_cigar.md [hunk 1]
```diff
@@ -7,4 +7,5 @@
 πως είναι ώρα να βγεις από εκεί. Πόρτα δεν υπάρχει οπότε πρέπει
 να χρησιμοποιήσεις κάτι από αυτά που έχεις. Τι κάνεις;
 
-[Ανάβεις τον αναπτήρα](set_it_on_fire/set_it_on_fire.md)
\ No newline at end of file
+[Ανάβεις τον αναπτήρα](set_it_on_fire/set_it_on_fire.md)
+[Πυρώνεις το πιρούνι με τον αναπτήρα](fork_it/fork_it.md)
```
Comment (casey-collab) [inline greek/offer_marshmallow_a_cigar/offer_marshmallow_a_cigar.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**